### PR TITLE
let medibot express itself better

### PR DESCRIPTION
**Pull request by Ilya246**
## About the PR
see https://github.com/space-wizards/space-station-14/pull/32698, exact copy except adapted for goob, wizden denied my PR but i thought you might like it better

## Why / Balance
funny and surprises you

## Media
see wiz PR

## Requirements
- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

**Changelog**
there's none sorry i was hungry

## Summary by CodeRabbit

- **New Features**
	- Introduced a new advertisement for the Medibot, enhancing the variety of ads displayed.
- **Updates**
	- Updated the count of Medibot advertisements to reflect the new entry, ensuring accurate representation in the system.

### CODE REVIEW

File: Resources/Prototypes/Catalog/VendingMachines/advertisements.yml [hunk 1]
```diff
@@ -266,4 +266,5 @@
   id: MedibotAds
   values:
     prefix: advertisement-medibot-
-    count: 17
+    # Goobstation - Medibot AM advertisement; originally 17
+    count: 18
```
[the remark this answers is not in the document]
Comment (deltanedas) [reply Resources/Prototypes/Catalog/VendingMachines/advertisements.yml:270]: inappropriate language ban

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Catalog/VendingMachines/advertisements.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Ilya246) [reply Resources/Prototypes/Catalog/VendingMachines/advertisements.yml:270]: > inappropriate language ban

banned for swore (hate bad word)

**Comment by Aidenkrz** [comment]
I feel like this has to high of a chance for it to say it, would be cool if you could find a way to give it an actual probability, or just add more lines for it to say.

**Comment by Ilya246** [comment]
> I feel like this has to high of a chance for it to say it, would be cool if you could find a way to give it an actual probability, or just add more lines for it to say.

like 45% per round, i calculated

**Review by whateverusername0 — APPROVED** [review]

**Comment by whateverusername0** [comment]
fixz checks

**Comment by whateverusername0** [comment]
@Ilya246

**Comment by Ilya246** [comment]
> fixz checks

says something about game maps so not my fault

**Comment by whateverusername0** [comment]
> says something about game maps so not my fault

true

**Pull request merged**